Makefile

```diff
@@ -116,7 +116,7 @@ BUILD_DEP_HASHES: $(BUILD_DEP_FILES)
 clang.bazelrc: bazel/setup_clang.sh
 	$(call install_clang)
 	bazel/setup_clang.sh /usr/lib/llvm-17
-	echo "build --config=libc++" >> $@
+	echo "build --config=clang" >> $@
```

Comment:
nice, I noticed similar thing locally as well, the config is actually simplified in upstream

https://github.com/envoyproxy/envoy/releases/tag/v1.35.0
> Consolidated clang/gcc toolchains using --config=clang or --config=gcc.